the-events-calendar.php

```diff
@@ -73,4 +73,4 @@ function tribe_events_not_php_version_plugin_name( $names ) {
 Tribe__Events__Main::instance();
 
 register_activation_hook( TRIBE_EVENTS_FILE, array( 'Tribe__Events__Main', 'activate' ) );
-register_deactivation_hook( TRIBE_EVENTS_FILE, array( 'Tribe__Events__Main', 'deactivate' ) );
+register_deactivation_hook( TRIBE_EVENTS_FILE, array( 'Tribe__Events__Main', 'deactivate' ) );
```

Comment:
Not a problem, but I'd avoid this kind of thing in commits. There's not real change here - just thrash due to your IDE removing the end line?

Reply:
Hi @Camwyn I had a situation where the tests weren't running because I only made changes to the CSS file. I removed the extra line intentionally to trigger the tests. What's the recommended way to trigger tests in this scenario?